File: gluecodium/src/test/resources/smoke/comments/output/android/com/example/smoke/CommentsMarkdown.java
/*
 *
 */
package com.example.smoke;
import com.example.NativeBase;
/**
 * <p>First line.
 * Second line.
 * <p>Another paragraph. <strong>bold</strong> and <em>italic</em> and <code>code</code>.
 * <blockquote>
 * <p>blockquote
 * </blockquote>
 * <h1>Heading one</h1>
 * <h2>Heading two</h2>
 * <h3>Heading three</h3>
 * <p>Unordered list:
 * <ul>
 * <li>A</li>
 * <li>B</li>
 * </ul>
 * <p>Ordered list:
 * <ol>
 * <li>foo</li>
 * <li>bar</li>
 * </ol>
 * <hr />
 * <p><a href="https://www.markdownguide.org/cheat-sheet/">title</a>
 */
public final class CommentsMarkdown extends NativeBase {
    /**
     * For internal use only.
     * @hidden
     * @param nativeHandle The SDK nativeHandle instance.
     * @param dummy The SDK dummy instance.
     */
    protected CommentsMarkdown(final long nativeHandle, final Object dummy) {
        super(nativeHandle, new Disposer() {
            @Override
            public void disposeNative(long handle) {
                disposeNativeHandle(handle);
            }
        });
    }
    private static native void disposeNativeHandle(long nativeHandle);
}

File: gluecodium/src/test/resources/smoke/comments/output/android-kotlin/com/example/smoke/CommentsMarkdown.kt
/*

 *
 */

@file:JvmName("CommentsMarkdownExtensions")

package com.example.smoke

import com.example.NativeBase

/**
 * First line.
 * Second line.
 *
 * Another paragraph. **bold** and *italic* and `code`.
 *
 * > blockquote
 *
 * # Heading one
 *
 * ## Heading two
 *
 * ### Heading three
 *
 * Unordered list:
 * - A
 * - B
 *
 * Ordered list:
 * 1. foo
 * 2. bar
 *
 * ---
 *
 * [title](https://www.markdownguide.org/cheat-sheet/)
 */
class CommentsMarkdown : NativeBase {



    /**
     * For internal use only.
     * @suppress
     * @param nativeHandle The handle to resources on C++ side.
     * @param tag Tag used by callers to avoid overload resolution problems.
     */
    protected constructor(nativeHandle: Long, @Suppress("UNUSED_PARAMETER") tag: Any?)
        : super(nativeHandle, { disposeNativeHandle(it) }) {}







    companion object {
        @JvmStatic private external fun disposeNativeHandle(nativeHandle: Long)
    }
}

